packages/api-apw/src/plugins/utils.ts

```diff
@@ -173,6 +170,6 @@ export const isInstallationPending = ({
     if (!tenant) {
         return true;
     }
-    const locale = i18nContent.locale;
-    return !locale;
+
+    return !i18n.getContentLocale();
```

Comment:
I do prefer the structure of dropping it into a variable, and then returning that variable. Otherwise it's doing too much in my opinion.

Reply:
If you look closer, you'll notice that I'm replacing the old `i18nContent.locale` usage to `i18n.getContentLocale()`. That was the primary objective of this change. Converting it into one liner was just a natural next step. The resulting code is really common. Assigning to variable just to check if it's a `!locale` is a matter of preference. I usually try to avoid a new variable for obvious statements.